src/lib/user-ip-address.js

```diff
@@ -0,0 +1,17 @@
+const net = require("net");
+
+module.exports = (request) => {
```

Comment:
We should pass it just the `forwarded` header so the logic shrinks down

Reply:
By sending request we can change logic in later time with out affecting other parts of the code.